### PR TITLE
Add Indonesia Convert Cup 2020 wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
wiki article for the recently concluded Indonesia Convert Cup 2020 tournament

### CODE REVIEW

File: wiki/Tournaments/ICC/2020/en.md [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+tags:
+  - ICC2020
+  - ICC 2020
+---
+
+# Indonesia Convert Cup 2020
+
+![ICC 2020 logo](img/logo.jpg)
+
+The **Indonesia Convert Cup 2020** (***ICC 2020***) was a double-elimination 1v1 osu!catch tournament hosted by ![][flag_ID] [Eum](https://osu.ppy.sh/users/16552751), ![][flag_ID] [Kazuyo](https://osu.ppy.sh/users/5604201), and ![][flag_ID] [Dapuluous](https://osu.ppy.sh/users/8140944) in where all maps in the tournament were converted osu!standard maps. The tournament was open to all osu!catch players from Indonesia with over 5,000 osu!catch playcounts regardless of rank. It was the first iteration of the tournament series.
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:11]:
```suggestion
The **Indonesia Convert Cup 2020** (***ICC 2020***) was a double-elimination 1v1 osu!catch tournament hosted by ![][flag_ID] [Eum](https://osu.ppy.sh/users/16552751), ![][flag_ID] [Kazuyo](https://osu.ppy.sh/users/5604201), and ![][flag_ID] [Dapuluous](https://osu.ppy.sh/users/8140944) in which, all beatmaps played in the tournament were converted osu! beatmaps. The tournament was open to all osu!catch players from Indonesia with over 5,000 osu!catch playcount regardless of rank. It was the first installment of the Indonesia Convert Cup.
```

File: wiki/Tournaments/ICC/2020/en.md [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2020-09-13/2020-09-23 |
+| Screening phase | 2020-09-24/2020-09-30 |
+| Group Stage | 2020-10-05/2020-10-11 |
+| Round of 16 | 2020-10-12/2020-10-18 |
+| Quarterfinals | 2020-10-19/2020-10-25 |
+| Semifinals | 2020-10-26/2020-11-01 |
+| Finals Week | 2020-11-02/2020-11-08 |
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:23]:
```suggestion
| Finals | 2020-11-02/2020-11-08 |
```

File: wiki/Tournaments/ICC/2020/en.md [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | IDR 1.000.000,- + 50% of the raised prize pool, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | IDR 600.000,- + 30% of the raised prize pool |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | IDR 450.000,- + 20% of the raised prize pool |
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:31]:
```suggestion
| Placing | Prizes |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | IDR 1.000.000, ~50% of the raised prize pool, unique profile badge |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | IDR 600.000, ~30% of the raised prize pool |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | IDR 450.000, ~20% of the raised prize pool |
```

File: wiki/Tournaments/ICC/2020/en.md [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+
+![Profile badge](img/badge.png)
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:33]:
```suggestion
![ICC 2020 winner badge](img/badge.png)
```

File: wiki/Tournaments/ICC/2020/en.md [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+
+## Organization
+
+ICC 2020 was run by various osu! community members predominantly hailing from Indonesia.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ![][flag_ID] [Eum](https://osu.ppy.sh/users/16552751), ![][flag_ID] [Kazuyo](https://osu.ppy.sh/users/5604201), ![][flag_ID] [Dapuluous](https://osu.ppy.sh/users/8140944) |
+| Map selector | ![][flag_ID] [Constantine](https://osu.ppy.sh/users/3221898), ![][flag_ID] [Yuri-](https://osu.ppy.sh/users/4983719), ![][flag_ID] [Toka](https://osu.ppy.sh/users/1595221) |
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:42]:
```suggestion
| Mappool selector | ![][flag_ID] [Constantine](https://osu.ppy.sh/users/3221898), ![][flag_ID] [Yuri-](https://osu.ppy.sh/users/4983719), ![][flag_ID] [Toka](https://osu.ppy.sh/users/1595221) |
```

File: wiki/Tournaments/ICC/2020/en.md [hunk 6]
```diff
@@ -0,0 +1,406 @@
...
+| Graphic designer | ![][flag_ID] [mina-](https://osu.ppy.sh/users/4670692), ![][flag_ID] [Avoya](https://osu.ppy.sh/users/3136817) |
+| Spreadsheet manager | ![][flag_ID] [Eum](https://osu.ppy.sh/users/16552751), ![][flag_ID] [Dapuluous](https://osu.ppy.sh/users/8140944) |
+| Wiki editor | ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805) |
+
+## Links
+
+- **[Main spreadsheet](http://docs.google.com/spreadsheets/d/e/2PACX-1vTjldrvfwKxuc3x8QyhVAcccvGm6z2FDR-BZuxyX68GkQkdXoWwkcUlD0gaNA__aKJRlX3WSMfW5Pkj/pubhtml#)**
+- [Discussion thread](http://osu.ppy.sh/community/forums/topics/1141903)
+- [Discord server](http://discord.gg/7g6TAj2)
+- [Livestream](https://www.twitch.tv/osuIndonesia/)
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:55]:
```suggestion
- [Livestream](https://www.twitch.tv/osuIndonesia)
```

File: wiki/Tournaments/ICC/2020/en.md [hunk 7]
```diff
@@ -0,0 +1,406 @@
...
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Upper | [Shurelia](https://osu.ppy.sh/users/3807986) ![][flag_ID] | 4 | **7** | ![][flag_ID] **[Galaxi](https://osu.ppy.sh/users/2552435)** | [#1](https://osu.ppy.sh/community/matches/69381788) |
+
+Saturday, 7 November 2020:
+
+| Bracket | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | [Shurelia](https://osu.ppy.sh/users/3807986) ![][flag_ID] | 0 | **7** | ![][flag_ID] **[dika312](https://osu.ppy.sh/users/741613)** | [#1](https://osu.ppy.sh/community/matches/69445742) |
+
+Sunday, 8 November 2020, Grand Final:
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:220]:
```suggestion
Sunday, 8 November 2020 (Grand Finals):
```

File: wiki/Tournaments/ICC/2020/en.md [hunk 8]
```diff
@@ -0,0 +1,406 @@
...
+
+| Bracket | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Grand Final | [Galaxi](https://osu.ppy.sh/users/2552435) ![][flag_ID] | 3 | **7** | ![][flag_ID] **[dika312](https://osu.ppy.sh/users/741613)** | [#1](https://osu.ppy.sh/community/matches/69502037) |
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:224]:
```suggestion
| Grand Finals | [Galaxi](https://osu.ppy.sh/users/2552435) ![][flag_ID] | 3 | **7** | ![][flag_ID] **[dika312](https://osu.ppy.sh/users/741613)** | [#1](https://osu.ppy.sh/community/matches/69502037) |
```

File: wiki/Tournaments/ICC/2020/en.md [hunk 9]
```diff
@@ -0,0 +1,406 @@
...
+| Grand Final (Bracket Reset) | **[dika312](https://osu.ppy.sh/users/741613)** ![][flag_ID] | **7** | 3 | ![][flag_ID] [Galaxi](https://osu.ppy.sh/users/2552435) | [#1](https://osu.ppy.sh/community/matches/69505076) |
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:225]:
```suggestion
| Grand Finals (Bracket Reset) | **[dika312](https://osu.ppy.sh/users/741613)** ![][flag_ID] | **7** | 3 | ![][flag_ID] [Galaxi](https://osu.ppy.sh/users/2552435) | [#1](https://osu.ppy.sh/community/matches/69505076) |
```

File: wiki/Tournaments/ICC/2020/en.md [hunk 10]
```diff
@@ -0,0 +1,406 @@
...
+   - In the Grand Finals, the winner of the the Upper Bracket only needs to win a single match against their opponent in order to claim the championship title. The winner of the Lower Bracket, however, needs to win two matches and enforce a Bracket Reset against their opponent in order to claim the championship title.
+4. Players who can compete in the next round are determined by:
+   - In the Round of 16, each player needs to win 4 points in order to win a match. (Best-of-7)
+   - In the Quarterfinals, each player needs to win 5 points in order to win a match. (Best-of-9)
+   - In the Semifinals, each player needs to win 6 points in order to win a match. (Best-of-11)
+   - In the Finals Week, each player needs to win 7 points in order to win the match. (Best-of-13)
+   - Player(s) who Win By Default.
+   - Whether there are player(s) who got disqualified during the match.
+
+### Match Regulations
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:394]:
```suggestion
### Match regulations
```

File: wiki/Tournaments/ICC/2020/en.md [hunk 11]
```diff
@@ -0,0 +1,406 @@
...
+4. Participants are **not allowed** to ban maps in the Group Stage.
+5. There is no Lower Bracket in the Group Stage.
+6. Rankings of each group are determined by sorting the results of each individual player's performance in the following priority:
+   - Most points obtained.
+   - Winner of the Tiebreaker.
+   - Player(s) who Win By Default.
+   - Whether there are player(s) who got disqualified during the match.
+7. Two players who placed on top of their respective groups' rankings based on the criteria listed above will advance to the knock-out stage.
+
+### Knock-Out Stage Rules
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:378]:
```suggestion
### Knockout Stage rules 
```

File: wiki/Tournaments/ICC/2020/en.md [hunk 12]
```diff
@@ -0,0 +1,406 @@
...
+
+1. All interested participants are required to register into the tournament individually.
+   - In order to be eligible to play in the tournament, a participant must have the Indonesian flag displayed on their profile and at least 5000 playcounts under their profile.
+2. To ensure valid and serious registrations, every registered participant will be checked by the Tournament Management and screened by osu!'s Tournament Staff.
+3. The list of participants who have passed the screening and are able to compete in the tournament will be published after the Registration Phase has ended.
+4. The Tournament Management will only account for a maximum of 32 participants to take part in the tournament.
+   - In the case where the number of concerned registrants exceeds 32, the Tournament Management will only enlist 32 registrants with the highest osu!catch pp under their account to participate as players in the tournament.
+5. Referees and map selectors may not participate as players in this tournament.
+
+### Group Stage Rules
```
Comment (TPGPL) [inline wiki/Tournaments/ICC/2020/en.md:361]:
```suggestion
### Group Stage rules
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Comment by Nivalyx** [comment]
applied most of TPGPL's suggestions

the only one that i didn't apply was the "osu!standard" -> "osu!" suggestion as i think "osu!standard" has more clarity for the context but aside from that everything else have been applied and resolved 👍

**Review by TPGPL — APPROVED** [review]

**Pull request merged**